src/Compilers/CSharp/Portable/Symbols/TupleTypeSymbol.cs

```diff
@@ -227,6 +225,13 @@ public override TResult Accept<TResult>(CSharpSymbolVisitor<TResult> visitor)
 
         internal override bool Equals(TypeSymbol t2, bool ignoreCustomModifiers, bool ignoreDynamic)
```

Comment:
I think ContainingSymbol should return ContainingSymbol of the underlying tuple type.